test/spec/modules/richaudienceBidAdapter_spec.js

```diff
@@ -206,6 +261,224 @@ describe('Richaudience adapter tests', function () {
     });
   });
 
+  describe('UID test', function () {
+    pbjs.setConfig({
+      consentManagement: {
+        cmpApi: 'iab',
+        timeout: 5000,
+        allowAuctionWithoutConsent: true
+      }
+    });
+    it('Verify build id5', function () {
+      DEFAULT_PARAMS_WO_OPTIONAL[0].userId = {};
+      DEFAULT_PARAMS_WO_OPTIONAL[0].userId.id5id = 'id5-user-id';
+
+      var request = spec.buildRequests(DEFAULT_PARAMS_WO_OPTIONAL, DEFAULT_PARAMS_GDPR);
+      var requestContent = JSON.parse(request[0].data);
+
+      expect(requestContent.user).to.deep.equal([{
+        'userId': 'id5-sync.com',
```

Comment:
I think these two fields are backwards. The `source` should be `'id5-sync.com'` and the `userId` that you set on line 274 is `'id5-user-id'`...so my guess is your tests aren't passing. Besides this everything else looks ok for ID5